go.mod

```diff
@@ -2,7 +2,7 @@ module github.com/bufbuild/buf
 
 go 1.21.0
 
-toolchain go1.23.0
+toolchain go1.21.13
```

Comment:
This kind of stinks - we want this to remain 1.23 - @emcfarlane whenever you get a chance, can you look into this job to get it to run on 1.23 so this doesn't happen?